images/linux/scripts/installers/preimagedata.sh

```diff
@@ -19,7 +19,7 @@ cat <<EOF > $imagedata_file
     "detail": "${os_name}"
   },
   {
-    "group": "Virtual Environment",
+    "group": "Runner Image",
     "detail": "Environment: ${image_label}\nVersion: ${image_version}\nIncluded Software: ${software_url}\nImage Release: ${releaseUrl}"
```

Comment:
```suggestion
    "detail": "Label: ${image_label}\nVersion: ${image_version}\nIncluded Software: ${software_url}\nImage Release: ${releaseUrl}"
```


same comment for the other OSes